example/scalalib/native/1-simple/build.mill

```diff
@@ -0,0 +1,137 @@
+//// SNIPPET:BUILD
+package build
+import mill._, scalalib._, scalanativelib._
+
+object `package` extends RootModule with ScalaNativeModule {
+  def scalaVersion = "2.13.11"
+  def scalaNativeVersion = "0.5.5"
+
+  // You can have arbitrary numbers of third-party dependencies
+  def ivyDeps = Agg(
+    ivy"com.lihaoyi::fansi::0.5.0",
+    ivy"com.lihaoyi::mainargs::0.7.6"
+  )
+
+  object test extends ScalaNativeTests {
+    def ivyDeps = Agg(ivy"com.lihaoyi::utest::0.8.4")
+    def testFramework = "utest.runner.Framework"
+  }
+}
+
+// This is a basic Mill build for a single `ScalaNativeModule`, with two
+// third-party dependencies and a test suite using the uTest framework. As a
+// single-module project, it `extends RootModule` to mark ``object `package``` as the
+// top-level module in the build. This lets us directly perform operations
+// `./mill compile` or `./mill run` without needing to prefix it as
+// `foo.compile` or `foo.run`.
+//
+//// SNIPPET:TREE
+// ----
+// build.mill
+// src/
+//     Foo.scala
+//     ...
+// test/
+//     src/
+//         FooTests.scala
+// out/
+//     compile.json
+//     compile.dest/
+//     ...
+//     test/
+//         compile.json
+//         compile.dest/
+//         ...
+// ----
+//
+
+//// SNIPPET:END
+
+//
+// You can download this example project using the *download* link above
+// if you want to try out the commands below yourself. The only requirement is
+// that you have some version of the JVM installed; the `./mill` script takes
+// care of any further dependencies that need to be downloaded.
+//
+// The source code for this module lives in the `src/` folder.
+// Output for this module (compiled files, resolved dependency lists, ...)
+// lives in `out/`.
+//
+//// SNIPPET:DEPENDENCIES
+//
+//
+/** Usage
+
+> ./mill resolve _ # List what tasks are available to run
+assembly
+...
+clean
+...
+compile
+...
+run
+...
+show
+...
+inspect
+...
+
+> ./mill run --text hello
+...
+#1 [info] Linking native code (immix gc, none lto) (185 ms)
+#1 [info] Postprocessing (0 ms)
+#1 [info] Total (24202 ms)
+[90/1] run 
+<h1>hello</h1>

```

Comment:
The syntax here looks wrong; are the example tests actually being run?

Reply:
You might need to register this folder in the example/package.mill. Please follow the existing code that's already there, and follow instructions in the readme for how to run them, and verify thst you can run the tests and that they pass

Reply:
Okay